README.md

```diff
@@ -1,13 +1,24 @@
 # [hammer.js][hammerjs-url]  [![NPM Version][npm-image]][npm-url]  [![NPM Downloads][downloads-image]][downloads-url]  [![Build Status][travis-image]][travis-url]
 > A JavaScript library for detecting touch gestures.
 
-## Solved Issue
+## What is this repository?
+
+It seems like that `hammer.js` repo is no more acitvated. 
```

Comment:
It would be better to write only one of the **repo** and **repository**.

Reply:
@younkue I fix it to repository.